docs/deployments/environments.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Environments / Instances
+description: An overview of the differences between Development and Production instances
+---
+
+# Environments / Instances
+
+When creating a new application within Clerk, you are typically provided with two instances: `Development` and `Production`.  These instances vary slightly and should be used appropriately.
```

Comment:
> you are typically provided with two instances

Can we say "you are provided with two instances" instead? In what scenario would someone be provided with a different set of instances?

Reply:
Let's keep the language unambiguous and update if/when more instances types are supported independent of that integration 👍